src/NUnitEngine/nunit.engine/Services/ProjectLoaders/NUnitProject.cs

```diff
@@ -259,6 +254,20 @@ private string GetBasePathForConfig(XmlNode configNode)
             return settings;
         }
 
+        static readonly char[] PATH_SEPARATORS = new char[] { '/', '\\' };
+
+        private string NormalizePath(string path)
+        {
+            char sep = Path.DirectorySeparatorChar;
+
+            if (path != null)
+                foreach (char alt in PATH_SEPARATORS)
```

Comment:
This would be a good example for the standards doc. You could add one, two or three pairs of brackets to this. What's the optimum? Do you want to write it up for us all to review?